README.md

```diff
@@ -1,42 +1,20 @@
-# Clientside InstantSearch Demo
-## Introduction
+# Clientside Instantsearch Demo
 
-This Instantsearch based application demonstrates how to perform client-side only search using itemsjs. It uses [unplatform-io/clientside-instantsearch-itemsjs-adapter](https://github.com/unplatform-io/instantsearch-itemsjs-adapter) to connect Instantsearch to itemsjs.
+<img src="https://github.com/unplatform-io/instantsearch-itemsjs-adapter/blob/main/logo-orange.svg" alt="Instantsearch-ItemsJS-adapter" align="right" width="20%" />
 
-## Getting Started
+## Introduction Instantsearch-ItemsJS-adapter 
 
-To start the project run the following commands
+This Instantsearch based application demonstrates how to perform client-side search using the [Instantsearch-ItemsJS-adapter](https://github.com/unplatform-io/instantsearch-itemsjs-adapter) package. This package will connect Instantsearch with ItemsJS.
+
+## Use application
+### Visit website
+Go to [this link](https://clientside-instantsearch-demo.vercel.app/) to see the demo in action.
+
+### Run on localhost
+After cloning this GitHub repository, run the following commands to start the application on your localhost.
 
 ```bash
 npm install
 npm run dev
 ```
-
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
-
-## [Cypress](https://www.cypress.io/)
```

Comment:
I think its still relevant that automatic E2E test are available. Maybe add this in a Contribute section?